Extending/Packages/Installing-And-Uninstalling-Packages/index.md

```diff
@@ -0,0 +1,142 @@
+---
+versionFrom: 9.0.0
+versionTo: 10.0.0
+meta.Title: "Installing and uninstalling Umbraco packages"
+meta.Description: "How to install a package, and remove a package from a solution"
+---
+
+# Installing packages
+
+In the Umbraco Backoffice, there is a **Packages** section which can be used to browse community-made packages for the CMS. From version 9 of Umbraco, it is only possible to add packages to via NuGet.
+
+![Backoffice - Packages section](images/backoffice-packages-section.png)
+
+Navigating to a specific package in the section will present you with an overview of the package, as well as an install snippet for NuGet CLI.
+
+![Backoffice - Starter Kit package](images/backoffice-packages-section-package.png)
+
+The packages can be installed by using:
+- NuGet Package Manager in Visual Studio
+- Package Manager Console in Visual Studio
+- .NET CLI (usually accessible from the terminal/command prompt of your system)
+
+For example, to install TheStarterKit package for the CMS the command would be:
+
+`dotnet add package Umbraco.TheStarterKit`
+
+Navigating to the Package Manager in Visual Studio is more visual, and makes it easier to get an overview of already installed packages.
+
+![Visual Studio - nuget packages manager](images/nuget-installing-options.png)
+
+The Package Manager has an integrated search function that allows you to find any public NuGet package and install it on the project.
+
+![Visual Studio - finding The Starter Kit](images/nuget-package-in-manager.png)
+
+Once the package has been installed, it should show up under the **Packages** section in the backoffice, under **Installed** tab.
+
+![Backoffice - installed packages](images/backoffice-installed-packages.png)
+
+# Uninstalling packages
+
+Uninstalling packages is not always as straightforward as installing them.
+
+In this section we will provide two examples on uninstalling a package - The Starter Kit and SEO Checker.
+
+## Uninstalling packages like The Starter Kit
+
+The Starter Kit provides you with a boilerplate solution to build upon - the package installs document types, templates, media, content, and everything else needed to set up a simple website. There's little custom code/functionality involved, and that is usually the case for such (starter kit/starter websites) packages.
+
+To uninstall a package, either run a command like
+
+`dotnet remove package Umbraco.TheStarterKit`
+
+in the CLI, or use the visual interface of NuGet Package Manager.
+
+![Visual Studio - uninstalling via Package Manager](images/uninstalling-via-nuget-package-manager.png)
+
+It is recommended to clean the solution after removing any package - this can be done by right-clicking the project in Visual Studio and choosing the *Clean* option, or using `dotnet clean` command.
```

Comment:
```suggestion
It is recommended to clean the solution after removing any package. This can be done by right-clicking the project in Visual Studio and choosing the *Clean* option, or using the `dotnet clean` command.
```